src/pylorax/__init__.py

```diff
@@ -153,7 +153,10 @@ def run(self, dbo, product, version, release, variant="", bugurl="",
             installpkgs=None,
             size=2,
             add_templates=None,
-            add_template_vars=None):
+            add_template_vars=None,
+            add_arch_templates=None,
+            add_arch_template_vars=None,
+            template_tempdir=None):
 
```

Comment:
This really is a terribly long list of parameters. Seems to me like we should probably start thinking about grouping some of them into namedtuples or something like that.